gems/aws-eventstream/spec/encoder_spec.rb

```diff
@@ -25,33 +25,25 @@ module EventStream
             expect(test_io.string.freeze).to eq(expectation)
           end
         end
-      end
-
-      describe '#encode error' do
 
-        it 'raises an error when payload exceeds' do
-          payload = double('payload', :length => 16777217)
+        it 'encodes large payloads' do
```

Comment:
This is what I wanted, but I thought Alex wanted to keep tests around that assert no error raised?

Reply:
Ah - I think maybe we had a miscommunication on it.  I thought you didn't want to test these cases at all rather than just removing the expect not_to raise.